src/main.rs

```diff
@@ -20,6 +21,8 @@ use tracing_subscriber::layer::SubscriberExt;
 use tracing_subscriber::EnvFilter;
 
 const VERSION: &str = env!("CARGO_PKG_VERSION");
+const IMDS_HTTP_TIMEOUT_SEC: u64 = 5 * 60;
```

Comment:
Because these are only used in a single place, do we need to store these as variables?

Reply:
Done